### DISCUSSION TITLE
Couleur de fond des boutons de confirmation de l'acteur 'Détail de Reflet'

### DISCUSSION THREAD

**Discussion by JeremieDENAEGEL** [witsa/synapps#615] [Q&A] [answered]
Bonjour,

Est-ce-que quelqu'un a réussi à changer le fond de couleur des boutons de confirmation de l'acteur "Détail de Reflet" ?
Je voudrais qu'il soit en fond blanc pour améliorer le rendu

![1](https://github.com/witsa/synapps/assets/50445069/b5daef72-6ebd-4955-80f6-1fd21a18bea9)

Merci d'avance ;)

**Comment by Raphwit** [comment]
Hello, si tu souhaites changer la couleur de fond de tes boutons je peux te suggérer l'approche suivante :

-> On ajoute un acteur html dans notre scène
-> Son contenu doit être : 
```
<style> 
.myClass button {
background-color : white !important;
}
</style>
```

-> il n'y a plus qu'à donner la classe "myClass" avec ce nom ou autre à mon acteur détail de reflet.

-> Ca marche !!

On peut également mettre la couleur en paramètre sur notre acteur HTML pour pouvoir la changer plus facilement si jamais.
Voilà en espérant t'avoir aidé.

**Reply by JeremieDENAEGEL** [reply, marked as answer]
Hi,

Je viens te tester sur ma synapps.
Ca marche niquel ;)
Merci

Petite information, pour une vue téléphone, la taille du texte de l'acteur 'Détails de Reflet' doit être en 'px' et non en 'vmin' sinon le texte dans le bouton ne se centre pas correctement.
